### PR TITLE
STSWENG-125: API's for View Product Page

### PR DESCRIPTION
**Pull request by alain-cheng**

### CODE REVIEW

File: api/products/getReviews.ts
Comment (kayecandy) [inline api/products/getReviews.ts:1]: Let's rename this to 

`api/products/reviews.ts`

File: api/products/view/[productId].ts
Comment (kayecandy) [inline api/products/view/[productId].ts:1]: Rename to

`api/products/[productId]/index.ts`

File: api/reviews/[productId].ts
Comment (kayecandy) [inline api/reviews/[productId].ts:1]: Rename to

`api/products/[productId]/reviews`

**Review by kayecandy — CHANGES_REQUESTED** [review]

**Comment by alain-cheng** [comment]
@kayecandy changes addressed :)

**Review by kayecandy — APPROVED** [review]

**Pull request merged**